### PR TITLE
Slack: fixed loads of non-strict json in blocks

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31661

## Screenshots
Failed test:
image

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/Slack/Integrations/Slack/Slack.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ def send_slack_request_sync(client: slack.WebClient, method: str, http_verb: str
                 response = client.api_call(method, http_verb='GET', params=body)
         except SlackApiError as api_error:
             response = api_error.response
-            headers = response.headers
+            headers = response.headers  # type: ignore[union-attr]
```
Comment (liorblob) [inline Packs/Slack/Integrations/Slack/Slack.py:272]: remove ignore

File: Packs/Slack/Integrations/Slack/Slack.py [hunk 2]
```diff
@@ -279,7 +279,7 @@ def send_slack_request_sync(client: slack.WebClient, method: str, http_verb: str
             raise
         break
 
-    return response
+    return response  # type: ignore[return-value]
```
Comment (liorblob) [inline Packs/Slack/Integrations/Slack/Slack.py:282]: remove ignore

File: Packs/Slack/Integrations/Slack/Slack.py [hunk 3]
```diff
@@ -306,11 +306,11 @@ async def send_slack_request_async(client: slack.WebClient, method: str, http_ve
         try:
             if http_verb == 'POST':
                 if file_:
-                    response = await client.api_call(method, files={"file": file_}, data=body)
+                    response = await client.api_call(method, files={"file": file_}, data=body)  # type: ignore[misc]
```
Comment (liorblob) [inline Packs/Slack/Integrations/Slack/Slack.py:309]: remove ignore

File: Packs/Slack/Integrations/Slack/Slack.py [hunk 4]
```diff
@@ -306,11 +306,11 @@ async def send_slack_request_async(client: slack.WebClient, method: str, http_ve
...
                 else:
-                    response = await client.api_call(method, json=body)
+                    response = await client.api_call(method, json=body)  # type: ignore[misc]
             else:
-                response = await client.api_call(method, http_verb='GET', params=body)
+                response = await client.api_call(method, http_verb='GET', params=body)  # type: ignore[misc]
```
Comment (liorblob) [inline Packs/Slack/Integrations/Slack/Slack.py:313]: remove ignore

File: Packs/Slack/Integrations/Slack/Slack.py [hunk 5]
```diff
@@ -306,11 +306,11 @@ async def send_slack_request_async(client: slack.WebClient, method: str, http_ve
         try:
             if http_verb == 'POST':
                 if file_:
-                    response = await client.api_call(method, files={"file": file_}, data=body)
+                    response = await client.api_call(method, files={"file": file_}, data=body)  # type: ignore[misc]
                 else:
-                    response = await client.api_call(method, json=body)
+                    response = await client.api_call(method, json=body)  # type: ignore[misc]
```
Comment (liorblob) [inline Packs/Slack/Integrations/Slack/Slack.py:311]: remove ignore

File: Packs/Slack/Integrations/Slack/Slack.py [hunk 6]
```diff
@@ -2002,7 +2002,7 @@ def main():
         LOG(e)
         return_error(str(e))
     finally:
-        demisto.info(f'{command_name} completed. loop: {loop_info(CLIENT._event_loop)}')
+        demisto.info(f'{command_name} completed. loop: {loop_info(CLIENT._event_loop)}')  # type: ignore[arg-type]
```
Comment (liorblob) [inline Packs/Slack/Integrations/Slack/Slack.py:2005]: remove ignore

**Review by liorblob — APPROVED** [review]
Nice, just remove the ignores, the build will pass

File: Packs/Slack/Integrations/Slack/Slack.py (hunk 1 above)
Comment (jochman) [reply Packs/Slack/Integrations/Slack/Slack.py:272]: mypy falling on it

**Pull request merged**